applications/test/ImageContentTemplateTest.scala

```diff
@@ -1,18 +1,21 @@
 package test
+import scala.collection.JavaConverters._
 
 import org.scalatest.{DoNotDiscover, Matchers, FlatSpec}
 
 @DoNotDiscover class ImageContentTemplateTest extends FlatSpec with Matchers with ConfiguredTestSuite {
 
   it should "render a cartoon" in goTo("/commentisfree/cartoon/2013/jul/15/iain-duncan-smith-benefits-cap") { browser =>
     import browser._
-    el("[itemprop='headline']").text should be("Steve Bell on Iain Duncan Smith and the benefits cap – cartoon")
+    el("[itemprop='headline']").html().toString.trim should be(
+      "Steve Bell on Iain Duncan Smith and the benefits cap – cartoon",
+    )
     el(".media-primary img").attribute("src") should include("Steve-Bell-cartoon")
   }
 
   it should "render a picture" in goTo("/artanddesign/picture/2013/oct/08/photography") { browser =>
     import browser._
-    $("[itemprop='headline']").text should be("Early erotica - a picture from the past")
+    $("[itemprop='headline']").texts().asScala.mkString should be("Early erotica - a picture from the past")
```

Comment:
Not really sure why this is different to the first spec i.e. maybe we should just use `el(...).html()...` here too.

Reply:
fluentlenium has been updated and the interface is.... more picky, better. For instance with the case of 

```
- $("[itemprop='headline']").text should be("Early erotica - a picture from the past")
+ $("[itemprop='headline']").texts().asScala.mkString should be("Early erotica - a picture from the past")
```

The semantics of `text` was ambiguous and they introduced `texts: utils.List[String]`. In some occasions I used `.asScala.mkString`, and in others I used `.asScala.head`, both being of type `String`,  both doing the same thing, because the collection is of size 1 anyway. 

Similar business with 

```
- $("meta[name=thumbnail]").attribute("content") should include(
+ $("meta[name=thumbnail]").attributes("content").asScala.head
```

Because of the various ways I could have deal with those examples, and others, I decided that I should just focus on respecting the original spirit/intent of the test as much as possible even if it meant what appears to be an awkward syntax. This is particularly apparent with 

```
- $(".content__dateline time").text should be("Sun 19 Aug 2012 13.38 EDT")
+ $(".content__dateline time").texts().asScala.head should include("Sun 19 Aug 2012 13.38 EDT")
```   

where `should be` became `should include`, because html markup was standing in the way. I had an original version with matching the HTML markup itself (which would have been even more accurate), but again, I wanted to remain faithful to the spirit. 

Last but not least, with DCR coming to play, I decided that trying to be perfect on legacy, non DCR HTML stuff, wasn't worth it 🙂